Lib/test/test_email/test_headerregistry.py

```diff
@@ -1280,6 +1280,7 @@ def example_as_address(self, source, defects, decoded, display_name,
         # XXX: we have no comment support yet.
         #self.assertEqual(a.comment, comment)
 
+    @unittest.skipIf(is_wasi and Py_DEBUG, "stack depth too shallow in pydebug WASI")
```

Comment:
Does this work now the stack depth has been increased?

Reply:
I re-enabled them. Let's see what happens.

Reply:
Wow the WASI tests pass!